llvm/lib/Transforms/Instrumentation/AddressSanitizer.cpp

```diff
@@ -1568,6 +1572,15 @@ void AddressSanitizer::getInterestingMemoryOperands(
       break;
     }
     default:
+      if (auto *II = dyn_cast<IntrinsicInst>(I)) {
+        std::pair<std::optional<MemIntrinsicInfo>,
+                  std::optional<SmallVector<InterestingMemoryOperand, 1>>>
+            MemInfo;
+        MemInfo = TTI->getTgtMemIntrinsic(II);
+        if (MemInfo.second != std::nullopt)
```

Comment:
```suggestion
        if (!MemInfo.second.empty())
```
Check the SmallVector is empty or not.